src/style.css

```diff
@@ -282,6 +282,11 @@
   border-radius: 12px;
 }
 
+/* style given to the active toggle button */
+.widget-buttons-grid>flowbox>flowboxchild>.active {
+  background: @noti-bg-focus;
+}
+
```

Comment:
Let's change this to this:
```css
.widget-buttons-grid>flowbox>flowboxchild>button{
  border-radius: 12px;
}

/* style given to the active toggle button */
.widget-buttons-grid>flowbox>flowboxchild>button.toggle:checked {
}

.widget-buttons-grid>flowbox>flowboxchild>button:hover {
}
```

Leave it empty to create some contrast between the box behind and the buttons :)